ajax/libs/react-datagrid/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "react-datagrid",
+  "version": "3.0.0",
+  "filename": "index.js",
+  "description": "React DataGrid",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/zippyui/react-datagrid.git"
+  },
+  "keywords": [
+    "grid",
+    "datagrid",
+    "data-grid",
+    "widget",
+    "table",
+    "list",
+    "react-component",
+    "react"
+  ],
+  "author": {
+    "name": "ZippyUI",
+    "email": "contact@zippyui.com"
+  },
```

Comment:
Please change author to `Radu Brehar` as given in the package's [package.json](https://unpkg.com/react-datagrid@3.0.0/package.json).